### PR TITLE
v5.6 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Comment by cpanato** [comment]
@amyblais added

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,171 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.6
+
+Release date: 2018-12-16
+
+### Highlights
```
Comment (jasonblais) [inline source/administration/changelog.md]: Thoughts about matching the higlights with the higlights in the blog post, where appropriate (e.g. mobile doesn't belong here, but CLI commands could be added here)?

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,171 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+#### Database Changes
+
+ - ``ExpireAt`` column was added to the ``PluginKeyValueStore`` table.
+ - Migrated user's accepted terms of service data into a new table called ``UserTermsOfService``.
+ - Removed ``idx_users_email_lower``, ``idx_users_username_lower``, ``idx_users_nickname_lower``, ``idx_users_firstname_lower`` and ``idx_users_lastname_lower`` indexes.
+
+#### Websocket Event Changes
+
+### Known Issues
```
Comment (jasonblais) [inline source/administration/changelog.md]: Any new known issues were not fixing that are worth mentioning

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,171 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+#### Administration
+ - Terms of Service feature improvements.
+ - Removed "System Console > Plugins > Configuration" page and moved enabling plugins setting to the Management page.
+
+#### Enterprise Edition (E20)
+ - Data Retention was promoted out of beta.
+
+### Bug Fixes
+
+ - Fixed an issue where pinned post list refreshed when a user posted a new message.
```
Comment (jasonblais) [inline source/administration/changelog.md]: Hm, did we only fix one bug in v5.6? Were the others regressions and not affecting v5.5?

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,6 +4,169 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+#### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+
+ - Under ``"ServiceSettings"`` in ``config.json``:
+    - Added ``"TLSMinVer": "1.2"``, ``"TLSStrictTransport": false``, ``"TLSStrictTransportMaxAge": 63072000`` and ``"TLSOverwriteCiphers": []``, to enable push notifications to use TPNS by default.
+ - Under ``"ExperimentalSettings"`` in ``config.json``:
+    - Added ``"EnablePostMetadata": false``, to
```
Comment (jasonblais) [inline source/administration/changelog.md]: Can you help ask Eric about this?

File: source/administration/changelog.md [hunk 5]
```diff
@@ -4,6 +4,169 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+    - Added ``"TLSMinVer": "1.2"``, ``"TLSStrictTransport": false``, ``"TLSStrictTransportMaxAge": 63072000`` and ``"TLSOverwriteCiphers": []``, to configure TLS connection when not using a reverse proxy such as NGINX.
+ - Under ``"ExperimentalSettings"`` in ``config.json``:
+    - Added ``"EnablePostMetadata": false``, to 
+
+### API Changes
+
+#### RESTful API v4 Changes
+ - Added ``/channels/{channel_id}/timezones`` to get a list of timezones for the users who are in the specified channel.
+
+//
```
Comment (jasonblais) [inline source/administration/changelog.md]: Jason to review list

File: source/administration/changelog.md [hunk 6]
```diff
@@ -4,6 +4,169 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+     - ``GetUsersInTeam``
+     - ``CreateDirectChannel``
+     - ``SearchChannels``
+     - ``SearchUsers`` 
+     - ``GetFileLink``
+     - ``UploadFile``
+     - ``SetProfileImage``
+
+#### Database Changes
+
```
Comment (jasonblais) [inline source/administration/changelog.md]: Jason to review list

File: source/administration/changelog.md [hunk 7]
```diff
@@ -4,6 +4,169 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Committs to migrate Mattermost Webapp to Redux.
+ - Idiomatic error handling
+
+#### User Interface
+ - Added ability to remove profile pictures in Account Settings.
+ - Added a new bar that shows progress on uploads.
+ - Added a new badge to the profile popover that indicates a user is a server admin.
+ - Added different experimental channel sidebar reorganization settings, such as the ability to sort channels by recent posts.
+ - Added an admin badge to profile popover to allow users to see if a user is a server admin.
+ - Added an option to be able to clear search results.
```
[the remark this answers is not in the document]
Comment (amyblais) [reply source/administration/changelog.md]: According to this it's webapp https://github.com/mattermost/mattermost-webapp/pull/1848

File: source/administration/changelog.md [hunk 8]
```diff
@@ -4,6 +4,169 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Enabled push notifications by default on new Mattermost installs, via an encrypted TPNS (test push notification service).
+ - Added a channel notification setting to disable @-channel @-here @-all notifications in specific channels.
+
+#### Performance
+ - Increased performance for returning user autocomplete results.
+
+#### Plugins
+ - Added "min_server_version" in plugin.json manifest to enable built-in control for preventing load/enable of plugin if server version is not met.
+ - Added ability for plugins to add channel header tooltips.
+ - Stop hashing plugin keys on write. **////XXXX**
```
Comment (amyblais) [inline source/administration/changelog.md]: For ABC @jasonblais

File: source/administration/changelog.md [hunk 9]
```diff
@@ -4,6 +4,169 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ #### ?
```
Comment (amyblais) [inline source/administration/changelog.md]: For ABC

**Pull request merged**